src/main/io/osd.c

```diff
@@ -135,12 +135,13 @@ typedef struct statistic_s {
     int16_t max_speed;
     int16_t min_voltage; // /10
     int16_t max_current; // /10
-    int16_t min_rssi;
+    int8_t min_rssi;
```

Comment:
This is wrong. `min_rssi` is set from `uint8_t getRssiPercent(void)`, so it needs to be `uint8_t` to avoid potential overflows.